### PR TITLE
replace "tech debt" with "long-term maintainability"

### PR DESCRIPTION
**Pull request by ericherman**
The use of the jargon term "technical debt" can be avoided. This is more clear English, and easier to translate.

See also:

https://github.com/standard-for-public-code/community-translations-standard/issues/48

 reported by @jgroenen

### CODE REVIEW

**Review by bvhme — APPROVED** [review]
This feels a lot clearer to me than previous. Also, I expect this is easier to act on for managers.

File: foreword.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Being open unlocks many other things.
 
 Local responsibility and democratic accountability are ensured when a public organization implements and maintains their own public code.
 By being open and with a broader contributor base, the software is more secure as it benefits from many eyes spotting potential flaws.
-Many contributors share the maintenance work to keep it functional and modern, which reduces future technical debt.
+Many contributors share the maintenance work to keep it functional and modern, which improves long-term maintainability.
```
Comment (ericherman) [inline foreword.md:96]: I wasn't confident that the foreword should change, but reading it now, I agree this is the correct call

**Review by Ainali — APPROVED** [review]
Good clarification!

**Review by bvhme — APPROVED** [review]

**Review by clausmullie — APPROVED** [review]

**Comment by ericherman** [comment]
There are four approvals on this, and no requests for change after a long waiting period.
I suggest this be merged.

**Pull request merged**